Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabDomUtils.cpp

```diff
@@ -26,6 +26,30 @@ namespace AzToolsFramework
     {
         namespace PrefabDomUtils
         {
+            namespace Internal
+            {
+                AZ::JsonSerializationResult::ResultCode JsonIssueReporter(AZStd::string& scratchBuffer,
+                    AZStd::string_view message, AZ::JsonSerializationResult::ResultCode result, AZStd::string_view path)
+                {
+                    namespace JSR = AZ::JsonSerializationResult;
+
+                    if (result.GetProcessing() == JSR::Processing::Halted)
+                    {
+                        scratchBuffer.append(message.begin(), message.end());
+                        scratchBuffer.append("\n    Reason: ");
+                        result.AppendToString(scratchBuffer, path);
+                        scratchBuffer.append(".");
+                        AZ_Warning("Prefab Serialization", false, "%s", scratchBuffer.c_str());
+
+                        scratchBuffer.clear();
+
+                        return JSR::ResultCode(result.GetTask(), JSR::Outcomes::PartialSkip);
```

Comment:
`Outcomes::Skipped` should be used instead of `Outcomes::PartialSkip`. The `Combine` function will automatically upgrade a `Skipped` to `PartialSkip` if another operation has completed, otherwise it will stay on `Skipped`.

Reply:
Done

Reply:
If you do this, then error reporting in best effort patching could be broken. We check whether patch application returned 'Outcomes::PartialSkip' in multiple places. 

But based on @AMZN-koppersr's explanation, outcome should be returned as 'partialskip' even if you provide 'skipped' here.

I recommend testing best effort patching after this change to 'skipped'. Open a prefab file and add an invalid patch to it. Load it in editor and see that error is reported that one of the patches couldn't be applied correctly.

Reply:
There was also a unit test with name 'ApplyPatch_UseJsonPatchWithCustomReportingCallback_ReportPartialSkip' added to test best effort patching. I recommending changing that also to 'skipped' to be consistent.

Reply:
Unit Test continues to work fine and should expect to receive PartialSkip based on the input Json.